src/MagnumPlugins/ObjImporter/Test/Test.cpp

```diff
@@ -702,6 +722,31 @@ void ObjImporterTest::wrongTextureCoordinateIndexCount() {
     CORRADE_COMPARE(out.str(), "Trade::ObjImporter::mesh3D(): some texture coordinate indices are missing\n");
 }
 
+void ObjImporterTest::multiMaterialObject() {
+    ObjImporter importer;
+    CORRADE_VERIFY(importer.openFile(Utility::Directory::join(OBJIMPORTER_TEST_DIR, "multiMaterial.obj")));
+    CORRADE_VERIFY(importer.mesh3DCount() == 2);
+    CORRADE_VERIFY(importer.materialCount() == 2);
+
+    /* Everything should be parsed properly */
+    std::optional<MeshData3D> data = importer.mesh3D(0);
+    CORRADE_VERIFY(data);
+    CORRADE_COMPARE(data->primitive(), MeshPrimitive::Triangles);
+    CORRADE_COMPARE(data->positionArrayCount(), 1);
+    CORRADE_COMPARE(data->positions(0), (std::vector<Vector3>{
+        {Vector3(1.72414, 18.9233, -3.20162),
+         Vector3(2.74428, -0.499733, -3.50576),
+         Vector3(-1.92235, -0.846268, 2.9722),
+         Vector3(1.72414, 18.9233, -3.20162),
+         Vector3(-1.92235, -0.846268, 2.9722),
+         Vector3(2.43556, 18.8755, 2.23745)}
```

Comment:
`f` suffix please, then you can just use `{}` without the explicit class name ;)